sci-chemistry/autodock/autodock-4.2.6-r1.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools readme.gentoo-r1
+
+MY_PN="autodocksuite"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A suite of automated docking tools"
+HOMEPAGE="http://autodock.scripps.edu/"
+SRC_URI="http://autodock.scripps.edu/downloads/autodock-registration/tars/dist$(ver_rs 1- '')/${MY_P}-src.tar.gz"
+S="${WORKDIR}/src"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples openmp"
+# Tests require python2
+RESTRICT="test"
+
+# False positives caused by nested configure scripts
+QA_CONFIGURE_OPTIONS=".*"
+
+DISABLE_AUTOFORMATTING="yes"
+DOC_CONTENTS="
+The AutoDock development team requests all users to fill out the
+registration form at:
+
+http://autodock.scripps.edu/downloads/autodock-registration
+
+The number of unique users of AutoDock is used by Prof. Arthur J.
+Olson and the Scripps Research Institude to support grant
+applications.
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-buildsystem.patch
+)
+
+src_prepare() {
+	default
+
+	find autodock -type f -name '*.cc' -exec sed -i 's/register //' {} \; || die
```

Comment:
Meh. Please write a patch for this instead, and we can compress it if needed. I don't like doing these because they can easily have different effects in later versions.

Reply:
Looks like we are going to have to compress it indeed:
```
SizeViolation: 'files/autodock-4.2.6-drop-register-keyword.patch' exceeds 20.0 KiB in size; 35.8 KiB total
```